src/libs/mysqlHandler.go

```diff
@@ -908,24 +908,27 @@ func UpdateWorkloadProcessFileSetMySQL(cfg types.ConfigDB, wpfs types.WorkloadPr
 }
 
 // UpdateOrInsertKubearmorLogsSQLite -- Update existing log or insert a new log into DB
-func UpdateOrInsertKubearmorLogsMySQL(cfg types.ConfigDB, kubearmorlogs []types.KubeArmorLog) error {
+func UpdateOrInsertKubearmorLogsMySQL(cfg types.ConfigDB, kubearmorlogmap map[types.KubeArmorLog]int) error {
 	db := connectMySQL(cfg)
 	defer db.Close()
 
-	for _, kubearmorlog := range kubearmorlogs {
-		if err := updateOrInsertKubearmorLogMySQL(db, kubearmorlog); err != nil {
+	start := time.Now().UnixNano() / int64(time.Millisecond)
```

Comment:
Instead of deriving, can we directly use time.Now().UnixMicro() for time in microseconds? or time.Now().Unix() for seconds.